message_sender/views.py

```diff
@@ -98,6 +98,14 @@ def get_serializer_class(self):
         return InboundSerializer
 
     def create(self, request, *args, **kwargs):
+        if ("duration" in
+                request.data.get("helper_metadata", {}).get("voice", {})):
+            fire_metric.apply_async(kwargs={
+                "metric_name": "vumimessage.obd.registration.length",
```

Comment:
Yes, I thought about this, is there any way to filter this to only get incoming registration calls?